core/scripts/umip-2/2_VoteSimulate.js

```diff
@@ -0,0 +1 @@
+../umip-3/2_VoteSimulate.js
```

Comment:
does this work to magically import the `2_VoteSimulate.js` script?

Reply:
It's just how github represents a symlink :)